### ISSUE TITLE
Use dynamically calculated average gas price over period of time in tx cost check for ticket face value

### ISSUE THREAD

**Issue by yondonfu** [livepeer/go-livepeer#2078]
Could use the eth_feeHistory JSON-RPC method.

See https://github.com/livepeer/go-livepeer/pull/2071 for background.

**Comment by leszko** [comment]
@ArcAster @yondonfu do you think it's still worth to implement this one? I have a feeling it became less important with Arbitrum (Nitro).

**Comment by yondonfu** [comment]
Agreed that this is less important with the current (and hopefully future post-Nitro) gas prices on Arbitrum. Let's ice box for now.
